tasks/main.yml

```diff
@@ -77,28 +123,28 @@
     - "{{ users }}"
     - public_keys
 
+# Now that we've hopefully configured some non-root users to be able to ssh
+# in, update the sshd config, which (among other things) will disable root
+# logins.
+- name: configure sshd
+  copy: src=sshd_config dest=/etc/ssh/sshd_config owner=root mode=0644
+  notify:
+    - restart sshd
+
 - name: check the active user accounts
   shell: "getent passwd $(basename -a /home/*) | cut -d: -f1"
   register: current_users_out
 
-- name: disable user accounts that are not listed devs or unmanaged accounts
+- name: disable user accounts that are not listed devs, unmanaged accounts, or project users
   user:
     name: "{{ item }}"
     state: absent
   with_items: "{{ current_users | difference(active_users) | difference(unmanaged_users) }}"
   vars:
-    current_users: "{{ current_users_out.stdout_lines | difference([project_user]) }}"
+    current_users: "{{ current_users_out.stdout_lines | difference([project_user])  }}"
```

Comment:
Nitpick: looks like an extra space snuck in, making it look like this line changed.